src/main/java/frc/robot/subsystems/CommandFactory.java

```diff
@@ -214,31 +214,51 @@ public Command algaeGoToL3() {
         .until(m_algaeRollers.hasAlgae());
   }
 
-  public Command algaeGoToL4() {
-    Command elevator = m_elevator.setCoralPosition(CoralLevels.L4);
+  public Command dealegfyL3() {
+    Command elevator = m_elevator.setCoralPosition(CoralLevels.DEALGEFY_L3);
     Command pivotAlgaeRollers = m_algaePivot.goToLowerSetpoint();
     Command IntakeAlgae = m_algaeRollers.feedIn();
 
     return elevator.andThen(pivotAlgaeRollers).alongWith(IntakeAlgae);
   }
 
+  public Command placeCoralL4() {
+    Command pivotCoralRollersCommand = m_coralPivot.goToLowerSetpoint();
+    Command ReleaseCoral = m_coralRollers.rollOutCommand(CoralLevels.L4);
+
+    return pivotCoralRollersCommand.andThen(ReleaseCoral);
+  }
+
   public Command putBallInProcesser() {
-    Command pivotAlgaeRollers = m_algaePivot.goToProssesorSetpoint();
+    Command pivotAlgaeRollers = m_algaePivot.goToMidPoint();
     Command finalCommand = pivotAlgaeRollers.andThen(m_algaeRollers.startFeedOut());
 
     return finalCommand;
   }
 
-  public Command placeCoral() {
+  public Command placeCoralL3() {
     Command pivotCoralRollersCommand = m_coralPivot.goToLowerSetpoint();
     Command ReleaseCoral = m_coralRollers.rollOutCommand(CoralLevels.L3);
 
     return pivotCoralRollersCommand.andThen(ReleaseCoral);
   }
 
+  public Command placeCoralL2() {
+    Command pivotCoralRollersCommand = m_coralPivot.goToLowerSetpoint();
+    Command ReleaseCoral = m_coralRollers.rollOutCommand(CoralLevels.L2);
+
+    return pivotCoralRollersCommand.andThen(ReleaseCoral);
+  }
+
```

Comment:
Since these commands are part of the auto, add the reuable suggested and then
```suggestion
  public Command placeCoralL2() {
    return placeCoral(CoralLevels.L2);
  }
```
And make same for other placeCoralLX commands.